prow/plank/reconciler.go

```diff
@@ -233,6 +257,16 @@ func (r *reconciler) syncClusterStatus(interval time.Duration, knownClusters map
 							r.log.WithField("cluster", cluster).WithError(err).Warn("Error listing pod to check for build cluster reachability.")
 							status = ClusterStatusUnreachable
 						}
+
+						// Additionally check for pod verbs.
+						if err := flagutil.CheckAuthorizations(client.ssar, r.config().PodNamespace, requiredTestPodVerbs); err != nil {
```

Comment:
We don't need to pass down `requiredTestPodVerbs` through the call stack like this. It can be a constant in the `plank` package that is referenced here and [in the prow-controller-manager's `main.go`](https://github.com/kubernetes/test-infra/pull/29282/commits/cfd0dc244139287bef3de9517ccd5b39acf43151#diff-94775c23a976e45bdafddb96bbc0a8091ac408c146989a9f95d8471dd41877a7R160-R170).

Reply:
Hmm while I agree with this idea, when I tried to implement it I got an import cycle.

But also, on second thought I think it makes sense to thread through the `requiredTestPodVerbs` to functions like `CheckAuthorizations` because that function is a generic function and is not plank-specific. So there's no avoiding passing it down the call stack.

Reply:
I've made a change locally and it avoids passing it down some parts, but not others.